config/build_simulation.py

```diff
@@ -0,0 +1,225 @@
+            for device in actor_row['devices']:
+                # save the csv file name
+                file_dict[device['deviceType']] = device['deviceID']
+
+        # Load
+        if 'load' in file_dict:
+            asset_dict['load'] = {"csv": loads_path.joinpath(file_dict['load']), "col_index": 1}
+
+        # PV
+        if 'solar' in file_dict:
+            asset_dict['pv'] = {"csv": pv_path.joinpath(file_dict['solar']), "col_index": 1}
+
+        # Prices
+        asset_dict['prices'] = {"csv": price_path.joinpath(price_filename), "col_index": 1}
+
+        actor = create_actor_from_config(actor_row['prosumerName'], asset_dict=asset_dict,
+                                         start_date=start_date, nb_ts=nb_ts, ts_hour=ts_hour,
+                                         ps=ps, ls=ls)
+
+        actors.append(actor)
+        print(f'{i} actor added')
+        print(f'{file_dict["load"]}')
+
+    actor_map = map_actors(config_df)
+    actor_map = pn.add_actors_map(actor_map)
+
+    if plot_network is True:
+        pn.plot()
+        pn.to_json()
+
+    # Update shortest paths and the grid fee matrix
+    pn.update_shortest_paths()
+    pn.generate_grid_fee_matrix(weight_factor)
+
+    return Scenario(pn, actors, actor_map)
+
+
+if __name__ == "__main__":
+    # Store default configuration files and directories paths
+    dirname = Path(__file__).parent
+    config_json_path = dirname / Path('community_config.json')
+    network_path = dirname / Path('example_network.json')
+    config_path = dirname / Path('config.txt')
+    loads_dir_path = dirname / Path('loads_dir.csv')
+    data_dirpath = Path("../sample")
+
+    # Parse configuration files and directories paths as arguments
+    parser = ArgumentParser(description='Entry point for market simulation')
+    parser.add_argument('scenario_config', nargs='?', default=config_json_path,
+                        help='scenario config json file')
+    parser.add_argument('network', nargs='?', default=network_path, help='network json file')
+    parser.add_argument('config', nargs='?', default=config_path, help='configuration file')
+    parser.add_argument('loads_dir', nargs='?', default=loads_dir_path,
+                        help='loads assignment csv file')
+    parser.add_argument('data_dir', nargs='?', default=data_dirpath, help='data directory')
+    args = parser.parse_args()
+
+    # Build config object using configuration file
+    cfg = Config(args.config)
+    cfg.path = Path('../') / cfg.path
+    # Reset save location directory
+    remove_existing_dir(cfg.path)
+    sc = create_scenario_from_config(args.scenario_config, args.network, data_dirpath=args.data_dir,
+                                     nb_ts=cfg.nb_ts, loads_dir_path=args.loads_dir, ps=1, ls=None)
+    sc.save(cfg.path, cfg.data_format)
+    update_actor_json_for_market_maker(cfg.path)
+
+    if cfg.show_plots:
+        sc.power_network.plot()
+        sc.plot_actor_data()
+
+    if cfg.save_network:
```

Comment:
should both be part of sc.save(), adapt function by:
- adding power_network.to_image()
- adapting code to use power_network.to_json() and implicit changes to the load function in scenario.py